src/main/config/config_master.h

```diff
@@ -86,6 +86,7 @@
 #define failsafeConfig(x) (&masterConfig.failsafeConfig)
 #define serialConfig(x) (&masterConfig.serialConfig)
 #define telemetryConfig(x) (&masterConfig.telemetryConfig)
+#define ibusTelemetryConfig(x) (&masterConfig.telemetryConfig)
```

Comment:
I added the field to the normal telemetry... until PGs... rather than create a whole new struct.